### PR TITLE
Replacement PR: Specify max tip on each tx 

### LINKED ISSUE #255: Supply gas hints to transactions in deployment script
## Why are we considering fiddling with the gas during deployment?
Ropsten deployment failed last week and it was gas related. The automatic gas fee hints that ethers.js was providing did not work. This _might_ be because Ropsten often has erratic gas prices, we're not certain.

However if we provide the gas hints at deployment time, we can check the prevailing price and add enough wiggle room that they'll likely all go through.

### PR DESCRIPTION
**Pull request by cliffhall**
## Replacement PR. Supersedes #256  

- In .env.example
  - Changed comment identifiers to # from // (this parsed by shell)
  - added `MAX_TIP`

- In deploy.ts,
  - in DeploymentExecutor
    - added `maxTip` and `txOptions`
    - in constructor
      - set maxTip to process.env.MAX_TIP
      - set txOptions to an object mapping `maxPriorityFeePerGas `to `this.maxTip`
    - in all deploy and method calls, pass `this.txOptions` as last or sole argument
        
- This fixes #255

### CODE REVIEW

**Comment by cliffhall** [comment]
This is a replacement for PR #256. I bungled the branch merging from main (there were conflicts so I tried to revert the merge and defer reconciliation but it didn't go so well) and and since there were only two files, I just made a new branch and replacement PR.

File: .env.example [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
-//DAI token Rinkeby address
+# DAI token Rinkeby address
 DAI_TOKEN=0x6A9865aDE2B6207dAAC49f8bCba9705dEB0B0e6D
 
-//metadata URIs
+# metadata URIs
 CONDITIONAL_COMMIT_TOKEN_METADATA_URI=changeme
-VOUCHERSETS_METADATA_URI=changeme
-VOUCHERS_METADATA_URI=changeme
+METADATA_BASE=changeme
```
Comment (hswopeams) [inline .env.example:18]: These still aren't correct. It should be 
OUCHERSETS_METADATA_URI=changeme
VOUCHERS_METADATA_URI=changme

METADATA_BASE should be removed

**Review by hswopeams — CHANGES_REQUESTED** [review]
See comment about .env.example

File: .env.example (hunk 1 above)
Comment (cliffhall) [reply .env.example]: Done.

**Review by hswopeams — DISMISSED** [review]

**Review by hswopeams — APPROVED** [review]

**Comment by hswopeams** [comment]
Approved, assuming tests will pass.

**Review by mischat — APPROVED** [review]
Let's do this.

**Pull request merged**